docs/IMAGEUPLOADER.md

```diff
@@ -45,3 +45,13 @@ Replace `XXXXXXXXXXXXXXX` with your API key from s-ul.eu. It can be found on [yo
 |Extra headers||
 |Image link|`{url}`|
 |Deletion link|`https://s-ul.eu/delete.php?file={filename}&key=XXXXXXXXXXXXXXX`|
+
+### [i.fourtf.com](https://github.com/fourtf/i)
+
+|Row|Description|
+|-|-|
+|Request URL|`https://i.yourwebsite.com/upload`|
+|Form field|`file`|
+|Extra headers|`Authorization: Basic <Your base64 encoded user and password>`|
```

Comment:
Replace `<Your base64 encoder user and password>` with something way shorter, like `XXXXXXXXXXXXXXX` (from an example above) with information to replace that with authentication.